lib/web.mlx

```diff
@@ -188,6 +188,75 @@ let getting_started ~install_url () =
 "$ cd hello_world"
 "$ dune pkg lock"
 "$ dune exec hello_world"
+    </Code>
+    <h3 class_="mt-10 mb-2.5">"Editor Configuration"</h3>
+    <p class_="text-white">
+    "Editor integration for OCaml is done using the tool "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    ". The recommended way of installing "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " is letting your editor run the provided "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " script. The installer places this script at ~/.dune/bin/ocamllsp. The script is deliberately named the same as the "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " executable so existing editor configurations will probably still work with the Dune Developer Preview (the exception is Visual Studio Code - see below)."
+    </p>
+    <p class_="text-white mt-4">
+    "Running "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " in a project with a dune.lock directory (generated by running "
+    <span class_="font-mono text-block-p">"dune pkg lock"</span>
+    ") will download and install "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " in the current project's _build directory, and then run it (yes that means that for now running "
+    <span class_="font-mono text-block-p">"dune clean"</span>
+    " will uninstall "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    "). If "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " is run elsewhere (outside a Dune project, or inside a Dune project lacking a dune.lock directory) then it will attempt to fall back to the installation of "
+    <span class_="font-mono text-block-p">"ocamllsp"</span>
+    " in the active Opam switch. This way you can continue to work on projects that manage their dependencies with Opam after installing the Dune Developer Preview without changing your editor configuration."
+    </p>
```

Comment:
My suggestion would be to wrap this in a `<Details />` section (see in the FAQ). One for VSCode and one for Neovim (we can add more later). It would make it clearer.